test/query_test.rb

```diff
@@ -50,4 +50,13 @@ def test_find_one!
     assert_equal user, User.find_one!(name: "Bryan")
     assert_raises(Minidoc::DocumentNotFoundError) { User.find_one!(name: "Noah") }
   end
+
+  def test_find_one_or_initialize
+    user = User.create(name: "Bryan")
+    assert user, User.find_one_or_initialize(name: "Bryan")
+    assert true, User.find_one_or_initialize(name: "Noah").is_a?(User)
+    assert true, User.find_one_or_initialize(name: "Noah").new_record?
```

Comment:
I believe the first argument is supposed to be the assertion, and the second argument is supposed to be a string to print if the first argument isn't truthy